app-admin/rcm/rcm-1.3.4.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
2019 -> 2020 :)

Reply:
fixed

Reply:
Use repoman to commit in future to avoid ;)